cfme/utils/appliance/__init__.py

```diff
@@ -511,6 +511,26 @@ def configure(self, log_callback=None, **kwargs):
 
             ipapp.wait_for_ssh()
 
+            # This is workaround for issue when httpd cannot start after restart because
```

Comment:
**REQUIRED:** Please move this out into a separate method and call that method from within the configure.